### PR TITLE
Create entities and migrates

### PR DESCRIPTION
**Pull request by vdumbrav**

### CODE REVIEW

File: src/Entity/Affiliate.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $active;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $created_at;
```
Comment (gregurco) [inline src/Entity/Affiliate.php]: camelCase

File: src/Entity/Job.php [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $how_to_apply;
```
Comment (gregurco) [inline src/Entity/Job.php]: also camelCase

File: src/Entity/Job.php [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $updated_at;
```
Comment (gregurco) [inline src/Entity/Job.php]: camelCase

File: src/Entity/Job.php [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $expires_at;
```
Comment (gregurco) [inline src/Entity/Job.php]: camelCase

**Review by gregurco — CHANGES_REQUESTED** [review]

**Comment by vdumbrav** [comment]
fix all snake_case to camelCase

**Pull request merged**